compressors/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+subprojects {
+  // https://github.com/gradle/gradle/issues/847
+  group = "io.opentelemetry.compressors"
```

Comment:
will this publish it under groupId `io.opentelemetry.compressors`?

Reply:
Oh this is something goofy. This just avoids a bug where nested modules have the same name. Like, if had `:foo:bar` and `:baz:bar`, there would be a conflict because some of the internals of gradle only identify the module by the last part of the module name, and we would have to modules named `bar`.

This still publishes under `io.opentelemetry.contrib`, but I think I can get rid of this actually because we don't have any other nested modules called `:zstd`. (In the core repo, we have multiple modules ending in `:testing-internal`.)